docs/joss/paper.bib

```diff
@@ -173,7 +173,7 @@ @misc{caffeine-site
 
 @misc{class-site,
   author={{Lawrence Berkeley National Laboratory}},
-  title = {{Computer Languages & Systems Software Group}},
+  title = {Computer Languages & Systems Software Group},
```

Comment:
This is the proper name of our [organizational group](https://go.lbl.gov/class), which is also always capitalized exactly as written (even outside of titles).

```suggestion
  title = {{Computer Languages & Systems Software Group}},
```